addons/mail/static/src/core/common/chat_hub.js

```diff
@@ -24,7 +24,7 @@ export class ChatHub extends Component {
         this.ui = useState(useService("ui"));
         this.bubblesHover = useHover("bubbles");
         this.moreHover = useHover(["more-button", "more-menu*"], () => {
-            this.more.isOpen = this.moreHover.isHover;
+            this.more.isOpen = this.moreHover.isHover || true;
```

Comment:
`|| true` makes the rest of the condition obsolete

Reply:
Oops, was there just to debug easily. Removing it